cpp/src/barretenberg/plonk/composer/composer_helper/composer_helper_lib.cpp

```diff
@@ -64,8 +61,12 @@ std::shared_ptr<plonk::verification_key> compute_verification_key_common(
             // Fetch the polynomial in its vector form.
 
             // Commit to the constraint selector polynomial and insert the commitment in the verification key.
-
-            auto poly_commitment = commitment_key.commit(proving_key->polynomial_store.get(poly_label));
+            auto polynomial = proving_key->polynomial_store.get(poly_label);
```

Comment:
This is how it used to be done until recently and reverting it was part of fixing Charlies PR. In principle the commitment key can be used but it would have to be plumbed in in a different way than was done here. This is Plonk only code and the commitment key (a Honk introduction) is generally not used anywhere else in Plonk. I generally stay away from refactoring honk code unless its absolutely necessary at this point.

Reply:
makes sense, thanks for explaining!